src/arkode/arkode_sprkstep.c

```diff
@@ -765,14 +765,14 @@ int sprkStep_AccessStepMem(void* arkode_mem, const char* fname,
   /* access ARKodeMem structure */
   if (arkode_mem == NULL)
   {
-    arkProcessError(NULL, ARK_MEM_NULL, "ARKODE::SPRKStep", fname,
```

Comment:
The input `fname` is no longer used and could be removed.

Reply:
This line youre referring to was a line that was removed.

Reply:
See this [comment](https://github.com/LLNL/sundials/pull/381#discussion_r1421126771)